pipeline/load.py

```diff
@@ -48,21 +48,29 @@ def insert_keywords(conn: connect, cursor: curs,
             conn.commit()
 
 
-def insert_keyword_recordings(conn: connect,
-                              cursor: curs, dataframe: pd.DataFrame) -> None:
-    """Inserts data into the keyword_recordings table"""
+def insert_keyword_recordings(conn: connect, cursor: curs, dataframe: pd.DataFrame) -> None:
```

Comment:
does this update the current hour?
so if one user entered a keyword at 3 15 and another user entered the keyword at 3 45, would the recordings update